sssd_test_framework/utils/local_override.py

```diff
@@ -0,0 +1,414 @@
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+
+__all__ = [
+    "LocalOverrideUtils",
+    "LocalOverrideUser",
+    "LocalOverrideGroup",
+]
+
+
+class LocalOverrideUtils(MultihostUtility[MultihostHost]):
```

Comment:
Perhaps rename all classes to SSSOverride* and file to sss_override.py since it is a wrapper around specific tool? What do you think is better?

Reply:
That makes more sense actually.